wp-content/themes/delta-child/page-about.php

```diff
@@ -60,46 +90,73 @@
+                        and talents to their fullest potential.
+                    </p>
+                    <p>
+                        The primary purpose of the Delta Program is to optimize the available study time for students,
+                        enabling them to undertake numerous long-term and short-term projects, often in
+                        collaboration with industry partners and startups. By doing so, the program not only enhances
+                        students' technical and professional capabilities but also prepares them for future
+                        challenges in their careers. The Delta program promotes self-directed learning, creativity, and
+                        a proactive mindset, ultimately aiming to produce graduates who excel in their
+                        fields and stand out in the competitive job market.
+                    </p>
+                </div>
+                <div class="col-lg-4 order-1 order-lg-1">
+                    <ul class="timeline">
+                        <li class="event" data-date="2005">
+                            <h3 style="color: #E5007D;" class="text-center-mobile">Delta Program launched</h3>
+                        </li>
+                        <li class="event" data-date="2012">
+                            <h3 class="text-center-mobile">Expansion to include students from other ICT profiles</h3>
+                        </li>
+                        <li class="event" data-date="2015">
+                            <h3 class="text-center-mobile">First successful startup from Delta</h3>
+                        </li>
+                        <li class="event" data-date="2016">
+                            <h3 class="text-center-mobile">First semester of Open Learning</h3>
+                        </li>
+                        <li class="event" data-date="2024">
+                            <h3 class="text-center-mobile" style="color: #E5007D;">Ongoing development</h3>
+                        </li>
+                    </ul>
+                </div>
+        </section>
+    </div>
+
+    <!-- Introduction to Delta -->
+    <section id="delta-activities" class="container pb-3 pb-lg-3">
+        <h2 class="justify-content-center d-flex pb-2 invisible-mobile">Introduction to Delta</h2>
+        <p class="justify-content-center text-center d-flex pb-4 mb-4 px-0 px-md-5 mx-0 mx-md-5 invisible-mobile">
+            The Fontys (ICT) Delta program offers students a unique opportunity, in addition to the regular study
+            program, to engage in various projects from Fontys or other external companies, resulting in an excellent
+            and diverse portfolio. With like-minded, motivated students, you acquire additional knowledge, skills and
+            experience reflecting the working environment, preparing you for work after your degree.
+        </p>
```

Comment:
Remove this text as the video will explain itself and to prevent too much text on the page.